WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardBloganuaryCardCell.swift

```diff
@@ -0,0 +1,138 @@
+import SwiftUI
+
+class DashboardBloganuaryCardCell: DashboardCollectionViewCell {
+
+    private var blog: Blog?
+
+    private lazy var cardFrameView: BlogDashboardCardFrameView = {
+        let frameView = BlogDashboardCardFrameView()
+        frameView.translatesAutoresizingMaskIntoConstraints = false
+        frameView.configureButtonContainerStackView()
+
+        // NOTE: this is intentionally called *before* setting the ellipsis button action,
+        // to avoid additional trailing padding.
+        frameView.hideHeader()
+
+        if let blog {
+            frameView.onEllipsisButtonTap = { }
+            frameView.ellipsisButton.showsMenuAsPrimaryAction = true
+            let action = BlogDashboardHelpers.makeHideCardAction(for: .bloganuaryNudge, blog: blog)
+            frameView.ellipsisButton.menu = UIMenu(title: String(), options: .displayInline, children: [action])
+        }
+
+        return frameView
+    }()
+
+    /// Checks whether the Bloganuary nudge card should be shown on the dashboard.
+    ///
+    /// The card is only going to be shown in December, and will be hidden in January.
+    /// It's also going to be shown for blogs that are marked as potential blogs by the backend, regardless
+    /// of whether the user has manually disabled the blogging prompts.
+    ///
+    /// - Parameters:
+    ///   - blog: The current `Blog` instance.
+    ///   - date: The date to check. Defaults to today.
+    /// - Returns: `true` if the Bloganuary card should be shown. `false` otherwise.
+    static func shouldShowCard(for blog: Blog, date: Date = Date()) -> Bool {
+        guard RemoteFeatureFlag.bloganuaryDashboardNudge.enabled(),
+              let context = blog.managedObjectContext else {
+            return false
+        }
+
+        // Check for date eligibility.
+        let isDateInDecember: Bool = {
+            let components = date.dateAndTimeComponents()
+            guard let month = components.month,
+                  let year = components.year else {
+                return false
+            }
+
+            // NOTE: For simplicity, we're going to hardcode the date check if the date is within December 2023.
+            // revisit this later so this doesn't have to be changed every year.
+            return month == 12 && year == 2023
```

Comment:
Could we just do `month == 12`? Or are we using `year == 2023` here because we don't know if we want this card next year?

Reply:
I was following the spec... However, now that I think about it, discarding the year check should make things a bit more resistant to change. I'll remove it along with fixing the multi-site issues above.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/22121/commits/c2447ca51cab02a24957b287882a7826789216f9.